packages/node/src/core/evm/abi-encoding.test.ts

```diff
@@ -0,0 +1,28 @@
+import { safeDecode } from './abi-encoding';
+
+describe('safeDecode', () => {
+  it('decodes parameters successfully', () => {
+    const res = safeDecode(
+      '0x315375000000000000000000000000000000000000000000000000000000000066726f6d0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000a0616d6f756e74000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000003e800000000000000000000000000000000000000000000000000000000000000034554480000000000000000000000000000000000000000000000000000000000'
+    );
+    expect(res).toEqual({
+      from: 'ETH',
+      amount: '1000',
+    });
+  });
+
+  it('returns null if parameters are empty', () => {
+    const res = safeDecode('0x');
+    expect(res).toEqual(null);
```

Comment:
Wouldn't `{}` be more accurate here?

Reply:
Yeah good point. I added some special cases for zero addresses to the airnode-abi package here: https://github.com/api3dao/airnode/pull/159/commits/db4adfe1257d39109d5f7ed0515474e17a54438b

I don't know if I need to cover both `AddressZero` and `HashZero` but it feels a bit safer

Reply:
I'm not sure about that. `AddressZero` and `HashZero` are not any different than `"this is not a valid ABI encoding"`.

Reply:
Ok so just handling `0x` is all we need then?

Reply:
I think so

Reply:
Decoding `AddressZero` or `HashZero` now throws an error